src/java/org/apache/cassandra/index/sai/disk/v1/bbtree/BlockBalancedTreeWriter.java

```diff
@@ -130,26 +132,26 @@ public int getMaxPointsInLeafNode()
     }
 
     /**
-     * Write the sorted values from a {@link BlockBalancedTreeIterator}.
+     * Write the sorted values from an {@link Iterator}.
      * <p>
      * @param treeOutput The {@link IndexOutput} to write the balanced tree to
-     * @param values The {@link BlockBalancedTreeIterator} containing the values and rowIDs to be written
+     * @param iterator The {@link Iterator} containing the values and rowIDs to be written
```

Comment:
It would be good to specify whether the index entries are expected to be sorted.

Reply:
I have updated this comment.